client/components/Application/Sidebar.tsx

```diff
@@ -0,0 +1,41 @@
+import Box from '@mui/material/Box'
+import Browser from './Browser'
+import LowerMenu from './LowerMenu'
+import { useTheme } from '@mui/material/styles'
+import sidebarLogo from '../../assets/ODE_sidebar_logo.svg'
+import Button from '@mui/material/Button'
+import * as store from '@client/store'
+
+export default function Sidebar() {
+  const theme = useTheme()
+  const contentHeight = `calc(100vh - ${theme.spacing(8 + 8 + 8)})`
```

Comment:
I like this implementation. It is simple.

Same with LowerMenu comment, have you tried: https://mui.com/material-ui/react-drawer/?

Looks like its the MUI component for side bars (that even provides a collapse function). The change is not mandatory, just wondering if you tried it.

Reply:
Hi, @pdelboca i also didn't know about this element, maybe we can switch to it if we decide to make the sidebar expandable.